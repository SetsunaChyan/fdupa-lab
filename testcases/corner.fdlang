function main(){
while (x >= 0) {
    x = x - 1;
    check_interval(x, 0, 0);
}
check_interval(x, 0, 255);
}