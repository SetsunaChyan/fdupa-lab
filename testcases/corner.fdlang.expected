Line 4: YES; x in [0, 0]
Line 6: Unreachable
